tensorflow_datasets/core/dataset_info.py

```diff
@@ -326,10 +339,10 @@ def _compute_dynamic_properties(self, builder):
     self._set_splits(splits)
 
   @property
-  def as_json(self):
+  def as_json(self) -> str:
     return json_format.MessageToJson(self.as_proto, sort_keys=True)
 
-  def write_to_directory(self, dataset_info_dir):
+  def write_to_directory(self, dataset_info_dir: str):
```

Comment:
`) -> None:` and similarly below (eg initialize_from_bucket)

Reply:
Done